packages/block-library/src/separator/theme.scss

```diff
@@ -8,4 +8,13 @@
 	&:not(.is-style-wide):not(.is-style-dots) {
 		max-width: 100px;
 	}
+
+	&.has-background:not(.is-style-dots) {
+		border-bottom: none;
```

Comment:
I think this `border-bottom: none` actually needs to be removed. Otherwise, I don't see any color for the default/wide variants in the front-end by default: 

**Editor**
<img width="628" alt="Screen Shot 2019-07-31 at 12 22 00 PM" src="https://user-images.githubusercontent.com/1202812/62229520-d1046e80-b38d-11e9-855c-2765c66799f7.png">


**Front end (Gutenberg Starter Theme)**
<img width="682" alt="Screen Shot 2019-07-31 at 12 16 37 PM" src="https://user-images.githubusercontent.com/1202812/62229534-d82b7c80-b38d-11e9-927d-eb32c01e77d9.png">

Reply:
this is weird, it should only be applied when there is a color, hence the `has-background`
Front end (Gutenberg Starter Theme)
![image](https://user-images.githubusercontent.com/6165348/62229740-4c7b1580-b3b8-11e9-829c-51468c34b725.png)

Reply:
Very weird! There must have been something wrong with my test post. Seems all set now, so please disregard this. 👍